src/v2/Apps/Order/Components/PriceOptions.tsx

```diff
@@ -30,6 +56,14 @@ export const PriceOptions: React.FC<PriceOptionsProps> = ({
   const [customValue, setCustomValue] = useState<number>()
   useEffect(() => {
     customValue && setValue(customValue)
+    // TODO: move this call if necessary once the feature is implemented
+    if (toggle && customValue && customValue < priceOptions[0]?.value!) {
+      tracking.trackEvent({
+        action_type: AnalyticsSchema.ActionType.ViewedOfferTooLow,
+        flow: AnalyticsSchema.Flow.MakeOffer,
+        order_id: order.internalID,
+      })
+    }
```

Comment:
This should trigger when the "Galleries usually accept offers within 20% of the listed price; any lower is likely to be rejected." notice is implemented on [NX-3013].

Reply:
if we're going to fix it soon I don't mind it here, otherwise, separate in a different useEffect call
```jsx
useEffect(() => {
    customValue && setValue(customValue)
}, [customValue, setValue])

useEffect(() => {
  // TODO: move this call if necessary once the feature is implemented
    if (toggle && customValue && customValue < priceOptions[0]?.value!) {
      tracking.trackEvent({
        action_type: AnalyticsSchema.ActionType.ViewedOfferTooLow,
        flow: AnalyticsSchema.Flow.MakeOffer,
        order_id: order.internalID,
      })
    }
}, [toggle, customValue, priceOptions, order])
```

Reply:
Sorry, I missed the other review but I just realized a naming Component design issue here.

The `setValue` prop should be called `onChange` to be more agnostic from the caller side in what to do when a change happens. `setValue` is too biased to what `OfferRoute` wants to do when a change happens.
 Can you change it, please? 💟

Reply:
Yeah, the `ViewedOfferTooLow` tracking event will be moved when that functionality is implemented.
Otherwise, added the dep update for `setValue` 👍

Reply:
thanks for doing it!
and about the other kind request? 😂 

>Sorry, I missed the other review but I just realized a naming Component design issue here.

>The setValue prop should be called onChange to be more agnostic from the caller side in what to do when a change happens. setValue is too biased to what OfferRoute wants to do when a change happens.
Can you change it, please? 💟

Reply:
Oops, sorry, I missed that. Fixed!